src/ConnectedMode.UnitTests/Migration/XmlDocumentHelperTests.cs

```diff
@@ -0,0 +1,133 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2023 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using SonarLint.VisualStudio.ConnectedMode.Migration;
+using SonarLint.VisualStudio.TestInfrastructure;
+
+namespace SonarLint.VisualStudio.ConnectedMode.UnitTests.Migration;
+
+[TestClass]
+public class XmlDocumentHelperTests
+{
+    [TestMethod]
+    public void MefCtor_CheckIsExported() =>
+        MefTestHelpers.CheckTypeCanBeImported<XmlDocumentHelper, IXmlDocumentHelper>();
+
+    [TestMethod]
+    public void MefCtor_CheckTypeIsShared() =>
+        MefTestHelpers.CheckIsSingletonMefComponent<XmlDocumentHelper>();
+    
+    [TestMethod]
+    public void LoadFromString_SaveToString_NoModification_ReturnsEquivalentString()
+    {
+        var input =
```

Comment:
Is it github formatting it like this? If not I'd consider fixing it a bit for readability

Reply:
I formatted it like this on purpose so that the multiline string indentation is consistent. However, in the other 2 tests something happened to the formatting